userprofile/templates/userprofile/profile.html

```diff
@@ -12,62 +12,60 @@
 {% endblock %}
 
 {% block content %}
-<div class="page-header">
-    <h1>
-        {% if user.first_name or user.last_name %}
-        {{ user.first_name }} {{ user.last_name }}
-        {% else %}
-        {% trans "[Unknown name]" %}
-        {% endif %}
-    </h1>
+<div class="row">
+	<div class="col-md-offset-2 col-md-2 page-header">
+		<div class="gravatar">
+			<a href="http://gravatar.com" target="_blank" rel="noopener noreferrer">
+				<img class="img-circle" src="{{ profile.avatar_url }}" alt="" aria-label="Change your avatar">
+				<span class="notify-badge">Change</span>
+			</a>
+		</div>
+		<h1>
+			{% if user.first_name or user.last_name %}
+			{{ user.first_name }} {{ user.last_name }}
+			{% else %}
+			{% trans "[Unknown name]" %}
+			{% endif %}
+		</h1>
+		<h3 style="margin-top: -10px;" class="text-muted">{% firstof user.username miss %}</h3>
+	</div>
 </div>
 <div class="row">
-    <div class="col-md-2">
-        <div class="thumbnail">
-            <img src="{{ profile.avatar_url }}" alt="avatar" />
-        </div>
-        <dl>
-            {% trans "[Missing]" as miss %}
-        	  <dt>{% trans "Student ID" %}</dt>
+    <div class="col-md-offset-2 col-xs-10 col-md-3">
+		<h2>Personal Information:</h2>
+		<dl class="dl-horizontal">
+			{% trans "[Missing]" as miss %}
+        	<dt>{% trans "Student ID" %}</dt>
             <dd>{% firstof profile.student_id miss %}</dd>
             <dt>{% trans "E-mail" %}</dt>
             <dd>{% firstof user.email miss %}</dd>
             <dt>{% trans "Username" %}</dt>
-            <dd>{% firstof user.username miss %}</dd>
-            <dt>{% trans "API Access Token" %}</dt>
-            <dd><input value="{{ profile.api_token }}" class="form-control"></dd>
-			<dt>{% trans "Language" %}</dt>
-			<dd>
-				<form action="{% url 'set-user-language' %}" method="post" class="form">{% csrf_token %}
-					<input name="next" type="hidden" value="{{ redirect_to }}">
-					<div class="input-group">
-						<select name="language" class="form-control">
-							{% get_current_language as LANGUAGE_CODE %}
-							{% get_available_languages as LANGUAGES %}
-							{% get_language_info_list for LANGUAGES as languages %}
-							{% for language in languages %}
-								<option value="{{ language.code }}" {% if language.code == LANGUAGE_CODE %}selected{% endif %}>
-									{{ language.name_local }}
-								</option>
-							{% endfor %}
-						</select>
-						<div class="input-group-btn">
-							<button type="submit" class="btn btn-primary">{% trans 'Save' %}</button>
-						</div>
-					</div>
-				</form>
-			</dd>
-        </dl>
+			<dd>{% firstof user.username miss %}</dd>
+		</dl>
+		<form class="form-inline profile-form">{% csrf_token %}
+			<div class="form-group">
+			  <label for="apiAccessToken">{% trans "API Access Token" %}</label>
```

Comment:
Have we used camelCase `apiAccessToken` or hyphenated `api-access-token` style in HTML ids before?

Hyphens are used in the style guide: https://apluslms.github.io/contribute/styleguides/html/